packages/cli/src/util/runner.ts

```diff
@@ -1,38 +1,40 @@
 import { getHttpOperationsFromResource } from '@stoplight/prism-http';
 import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
 import * as chokidar from 'chokidar';
-import { CreatePrismOptions } from './createServer';
+import * as os from 'os';
+import { CreateMockServerOptions } from './createServer';
 
-type CreatePrism = (options: CreatePrismOptions) => Promise<IPrismHttpServer | undefined>;
+type CreatePrism = (options: CreateMockServerOptions) => Promise<IPrismHttpServer | void>;
 
-export function runPrismAndSetupWatcher(createPrism: CreatePrism, options: CreatePrismOptions, spec: string) {
+export function runPrismAndSetupWatcher(createPrism: CreatePrism, options: CreateMockServerOptions) {
   return createPrism(options).then(possiblyServer => {
     if (possiblyServer) {
       let server: IPrismHttpServer = possiblyServer;
 
-      const watcher = chokidar.watch(spec, {
-        persistent: false,
+      const watcher = chokidar.watch(options.document, {
+        persistent: os.platform() === 'darwin',
```

Comment:
Can we get a comment here explaining what or why. Is this a temporary thing, is there a GitHub issue explaining a bug that caused this workaround, or documentation somewhere about why we are doing this? The reasoning could be forgotten over time.

Reply:
It's basically an OS-specific flag that we need to explicitly set, unfortunately. See https://github.com/paulmillr/chokidar#persistence

I'll add a comment.